core/primitives/impl/scale_codec_impl.cpp

```diff
@@ -336,6 +383,42 @@ namespace kagome::primitives {
     return decodeVariant<Invalid, Valid, Unknown>(stream);
   }
 
+  outcome::result<Buffer> ScaleCodecImpl::encodeInherentData(
+      const InherentData &inherentData) const {
+    auto &data = inherentData.getDataCollection();
+
+    // TODO(Harrm) PRE-153 Change to vectors of ref wrappers to avoid copying
+    // vectors
+    std::vector<InherentData::InherentIdentifier> ids;
+    ids.reserve(data.size());
+    std::vector<std::vector<uint8_t>> vals;
+    vals.reserve(data.size());
+
+    for (auto &pair : data) {
```

Comment:
```suggestion
    for (const auto &pair : data) {
```

Reply:
I think it's already const, and my IDE type deduction hints confirm it.